hail_search/fixtures/GRCh38/SNV_INDEL/annotations.ht/README.txt

```diff
@@ -1,3 +1,3 @@
 This folder comprises a Hail (www.hail.is) native Table or MatrixTable.
-  Written with version 0.2.128-eead8100a1c1
-  Created at 2024/06/14 15:14:52
\ No newline at end of file
+  Written with version 0.2.130-bea04d9c79b5
+  Created at 2024/08/30 15:20:20
```

Comment:
why is the annotations table changing?

Reply:
I added some variants for my new test cases